data/pilots/resistance/scavenged-yt-1300.json

```diff
@@ -63,7 +63,7 @@
       "initiative": 4,
       "limited": 1,
       "xws": "chewbacca-scavengedyt1300",
-      "ability": "After a friendly ship at range 0-3 is destroyed, before that ship is removed, you may perform an action. Then you may perform a bonus attack.",
+      "ability": "After a friendly ship at range 0-3 is destroyed, you may perform an action. Then you may perform a bonus attack.",
```

Comment:
The card does have that text:

![chewbacca](https://sb-cdn.fantasyflightgames.com/card_images/en/209a8dece475a265aaf1b5318ba7f08a.png)

Reply:
I don't see it in the errata either.

Reply:
Closing this as it's super old. Let me know if you think it should still be merged.